_info/Market

```diff
@@ -1,3 +1,5 @@
 Market
 
-Not to be mistaken for the Stock market, this market holds a Varity of useful items for anyone to use, which can drastically change the situation. But since there items are really powerful, there items are behind a heavy paywall. Use `!market` to check it out. You can use `!dismarket` to check your market level. You need to use `!upmarket` to upgrade your market. Use `!tmbuy` to buy the desired items. Note that Solos have the entire market unlocked for free. Also note that these items will only be used if you explicitly inform the hosts that you are using these items.
+Not to be mistaken for the Stock market, this market holds a Varity of useful items for anyone to use, which can drastically change the situation. But since there items are really powerful, there items are behind a heavy paywall. 
+Use `!market view` to check it out. You can use `!market level` to check your market level. You need to use `!upmarket` to upgrade your market. Use `!market buy` to buy the desired items. 
+Note that Solos have the entire market unlocked for free. Also note that these items will only be used if you explicitly inform the hosts that you are using these items.
```

Comment:
should mention that this is replaced with black market if dead

Reply:
Should be fixed.